test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -378,6 +378,37 @@ bool VerifyToastExpiresOnReboot()
     return true;
 }
 
+bool VerifyShowToast()
+{
+    // Registration happens in main()
+    winrt::ToastNotification toast{ GetToastNotification() };
+
+    auto toastNotificationManager = winrt::ToastNotificationManager::Default();
+    toastNotificationManager.ShowToast(toast);
```

Comment:
I think we can verify toast ID as well.

Reply:
Yes. Verify that ToastId != 0

Reply:
Ideally, we want to use the History API to ensure that the Post succeeded here. Can you add that as a ToDo comment here?

@loneursid  FYI